### ISSUE TITLE
Add documentation for processing multiplexed and non multiplexed assays in the same experiment

### ISSUE THREAD

**Issue by shanaxel42** [spacetx/starfish#1368]

**Comment by neuromusic** [comment]
Do we have any example data for this kind of experiment?

**Comment by neuromusic** [comment]
perhaps @mattcai & @berl can sync on this and figure out how we can provide documentation on how to handle this kind of experimental design

**Comment by Boehmin** [comment]
I`m probably stuck with the same issue, so I would be curious to see an example or how to solve this - has anyone attempted this so far?
Thank you!

**Comment by berl** [comment]
@Boehmin we did get this working on our internal Allen Institute pipeline that was built around `starfish`, but our use-case necessitated some complexity that you probably don't need.  The general idea is to create a sub codebook for the `(round, channel)` in the experiment that only have non-barcoded data and similarly subset the image data to only those relevant images. Then you can decode them separate from the the barcoded part of the experiment. If I recall correctly, the barcoded and non-barcoded results can be (easily?) combined after decoding.

**Comment by berl** [comment]
@Boehmin  No, we did all the codebook splitting in memory, loading the one codebook.json and subsetting it differently for barcoded and nonbarcoded rounds. Once it's loaded, you can identify rounds where `(r,c)` tuples have only one nonzero value and then you can subset the codebook to only these rounds. Similarly with barcoded rounds but with any `(r,c)` tuple being part of a codeword that has more than one nonzero value. If you have a round where some channels are used as part of a barcode and others are used as non-barcoded, it gets a bit more complicated, but is still possible

**Comment by Boehmin** [comment]
This makes so much more sense! Thanks for taking the time to explain - I`ll give this a go.
